boilerplate/package.json.ejs

```diff
@@ -62,8 +62,8 @@
 
     "@types/jest": "24.0.16",
     "@types/ramda": "0.26.18",
-    "@types/react": "16.8.23",
-    "@types/react-native": "0.60.2",
+    "@types/react": "16.8.24",
+    "@types/react-native": "0.60.3",
```

Comment:
Consider bumping to 0.60.5, which was released 3 days ago.

Reply:
Normally I'd avoid snowballing an RN upgrade in (I assume you meant that and not just the types!), but this was an easy one, so 👍

Reply:
Yeah -- just "consider", definitely always fine to tell me "not in this PR" ;)